packages/blockchain-wallet-v4-frontend/src/scenes/Register/index.tsx

```diff
@@ -20,6 +23,40 @@ class RegisterContainer extends React.PureComponent<PropsType, StateType> {
   state = {
     showForm: false
   }
+  componentDidMount () {
+    window.addEventListener('message', this.receiveMatomoMessage, false)
+    if (Remote.Success.is(this.props.abTest)) return
```

Comment:
can you put line 27 (`window.addEventListener('message', this.receiveMatomoMessage, false)`) after this line?  Just in case the `Remote` is already success, there is no need to add an event listener that will never trigger